### PR TITLE
Fixed https://github.com/SimformSolutionsPvtLtd/flutter_showcaseview/issues/313

### PR DESCRIPTION
**Pull request by kirill-21**
Added if(mounted) checks where error occured according to the PlatformDispatcher.instance.onError stackTrace

### CODE REVIEW

**Comment by ujas-m-simformsolutions** [comment]
@kirill-21 you have added `mouted` check where setState isn't being used so please remove them.

File: lib/src/layout_overlays.dart [hunk 1]
```diff
@@ -140,8 +140,11 @@ class _OverlayBuilderState extends State<OverlayBuilder> {
   @override
   void reassemble() {
     super.reassemble();
-    ambiguate(WidgetsBinding.instance)
-        ?.addPostFrameCallback((_) => syncWidgetAndOverlay());
+    if (mounted) {
```
Comment (ujas-m-simformsolutions) [inline lib/src/layout_overlays.dart:143]: For example here.

**Comment by kirill-21** [comment]
> @kirill-21 you have added `mouted` check where setState isn't being used so please remove them.

No, I tested this fix and added checks only when they are needed, problem is not only with setState method

**Review by ariefwijaya — APPROVED** [review]
LGTM!

**Pull request closed without merging**